db/sql/dbsetup.sql

```diff
@@ -305,6 +304,39 @@ CREATE INDEX endpoint_organization_url_idx ON endpoint_organization (url);
 CREATE INDEX vendor_id_idx ON vendors (id);
 CREATE INDEX fhir_endpoints_info_vendor_id_idx ON fhir_endpoints_info (vendor_id);
 CREATE INDEX fhir_endpoints_info_history_vendor_id_idx ON fhir_endpoints_info_history (vendor_id);
+CREATE INDEX vendor_name_idx ON vendors (name);
```

Comment:
I didn't find this index in any migration files, should this be added with your new migrations?

Reply:
Yeah you are right, I just added it in!